docs/interface.md

```diff
@@ -0,0 +1,56 @@
+# The Interface
+
+All interactions with the Exercism website are handled automatically. Test runners have the single responsibility of taking a solution, running its tests and returning a standardised output.
+
+## Execution
+
+- A test runner should provide an executable script. You can find more information in the [docker.md](https://github.com/exercism/automated-mentoring-support/blob/master/docs/docker.md) file.
+- The script will receive two parameters:
+  - The slug of the exercise (e.g. `two-fer`).
+  - A path to a directory containing the submitted file(s) (with a trailing slash).
+- The script must write a file to that directory named `report.json`
```

Comment:
Which directory?

Usually I'd say "that directory" refers to the second argument from the bullet point before, but as both bullet points are on a different level this feels wrong, and therefore I'd assume the working dir…

Could you elaborate/calrify?

Reply:
Could @ccare or someone from @exercism/website-copy-automated-comments who's done this for analysers reply to this pls, as I don't know :)

Reply:
IIRC we write to the inputdir, not the root dir. So "that" refers (correctly) back to "a path to a directory".

Reply:
> IIRC we write to the inputdir, not the root dir. 

This is correct. The result of an analyzer (`analysis.json`) is written to the same directory that contains the solution files.

Reply:
Thanks @ErikSchierboom 

Although the folder will be passed in as an arg (to aid loose coupling), it'll almost certainly be something like `/mnt/exercism/iteration` will be disposable after the run has completed. As well as being the target for output data it'll probably also be the only bit of read-write filesystem available. We will probably also provide a mutable, but ephemeral, /tmp